specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -363,6 +363,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/detailedusages": {

```

Comment:
/usages is a standard ARM contract to provide a consistent way across Azure to get usage related data. Let's not invent something new for this usecase. If the current usages contract is limiting and does not address your scenario, please start a thread on this with armapireview. 
Once we understand your scenario, there may be other ways to address your scenarios.